docs/application/dotnet/guides/overview.md

```diff
@@ -49,9 +49,9 @@ You can use the following TizenFX features in your .NET applications:
 
   The alarms and scheduling features introduce how you can define and store alarms.
 
-- [Media and Camera](multimedia/overview.md)
+- [Multimedia](multimedia/overview.md)
 
-  The media and camera features cover everything related to multimedia. You can record and play various media formats, use the device camera to take pictures, and listen to the radio. You can also handle media conversions and manage media streams.
+  Multimedia features cover everything related to multimedia. You can record and play various media formats, use the device camera to take pictures, and listen to the radio. You can also handle media conversions and manage media streams.
 
 - [Connectivity and Wireless](connectivity/overview.md)
 
```

Comment:
Line 66:
Please drop hyphen from “location-based”. Kindly rephrase as “location based”.


Line 90:
Kindly rephrase the line as “The natural language processing features explains…”

Reply:
I'm not in charge of those statements so can't modify those statements.